packages/build-utils/src/detect-builders.ts

```diff
@@ -278,7 +278,7 @@ export async function detectBuilders(
       // and package.json can be served as static files
       frontendBuilder = {
         use: '@vercel/static',
-        src: '!{api/**,package.json}',
+        src: '!{api/**,package.json,middleware.[jt]s}',
```

Comment:
What syntax is this? Glob pattern? I could be wrong, but i thought the right pattern would be `[j|t]s`

Reply:
These are matched with `minimatch`, so it's the syntax that that module uses. Essentially it's standard RegExp though:

```
> /middleware\.[jt]s/.test('middleware.js')
true
> /middleware\.[jt]s/.test('middleware.ts')
true
> /middleware\.[jt]s/.test('middleware.xs')
false
```